chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -66,11 +66,21 @@ if [[ -f /sbin/drbdadm ]] ; then
         log "Waiting for DRBD to bring the devices up..."
         sleep 1
     done
+    drbdadm status
+
+    # Wait for replication to start
+    while ! drbdadm status | grep -qi replication; do
+        log "Replication did not start yet: `drbdadm status`"
+        sleep 1
+    done
+    drbdadm status
+
     # Now wait after DRBD data are synced with the other node
-    while drbdadm status | grep -q Inconsistent; do
+    while drbdadm status | grep -qi -e inconsistent -e replication; do
```

Comment:
I don't know of sure but aren't we interesting in this:

```
postgresql role:Secondary
  disk:UpToDate
  ^^^^^^^^^^^^^
  d52-54-77-77-01-01 role:Primary
    peer-disk:UpToDate
```

AFAIK the first block of the ```drbdadm status``` output for a specific volume eflects the state of the local disk (i.e. the disk that the command is executed on).

ICBW though, and I guess it's better to clarify with @nick-wang .